.changeset/pink-yaks-jam.md

```diff
@@ -0,0 +1,5 @@
+---
+'vite-imagetools': patch
+---
+
+support Vite's base config when the dev server is running
```

Comment:
I think this would also affect `build` and not just `dev` since it changes code in `load`
```suggestion
fix: support Vite's `base` config
```

Reply:
`build` seems to work the same for me both before and after adding `base` support.